container-toolkit/versions.json

```diff
@@ -1,10 +1,13 @@
 {
-    "latest": "1.16.2",
+    "latest": "1.17.0",
     "versions":
     [
+        {
+            "version": "1.17.0"
+	    },
```

Comment:
Not technical, just curious that the editor didn't automatically indent evenly.

Reply:
Was a tabs vs spaces thing. Fixing.